tests/tests_suite_test.go

```diff
@@ -601,3 +605,25 @@ func TestFunctional(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecsWithDefaultAndCustomReporters(t, "Functional test suite", reporters)
 }
+
+func retrieveNodeArchitecture() string {
+	nodeList := &core.NodeList{}
+	err := apiClient.List(ctx, nodeList)
+	Expect(err).NotTo(HaveOccurred(), "Failed to list nodes")
+	nodes := nodeList.Items
+	Expect(nodes).NotTo(BeEmpty(), "No nodes found")
+	architecture := nodes[0].Status.NodeInfo.Architecture
+	Expect(architecture).To(BeElementOf([]string{"s390x", "amd64"}), "Unsupported architecture")
```

Comment:
With these expects the switch in `retrieveQemuMachineType` will never fallthrough to the default case. I know `q35` was hardcoded before. But what is the correct value to use on ARM64? Is it `virt`?

Reply:
@zhlhahaha Can you help with this?

Reply:
It is `virt` on Arm64.